components/textfield/package.json

```
@@ -58,6 +58,10 @@
 		{
 			"guidelines": "https://spectrum.adobe.com/page/text-field",
 			"rootClass": "spectrum-Textfield"
+		},
+		{
+			"guidelines": "https://spectrum.adobe.com/page/text-area",
+			"rootClass": "spectrum-Textarea"
```

```suggestion
			"rootClass": "spectrum-Textfield--multiline"
```

I guess technically this is the "rootClass" being used by the text area right?